### PR TITLE
Update substep_timings script

### PR DESCRIPTION
**Pull request by jgphpc**
- handle missing  "Upsweep" and "Gravity" timings

### CODE REVIEW

File: scripts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+matplotlib
+numpy
```
Comment (sekelle) [inline scripts/requirements.txt:2]: You could add h5py here.

Comment (jgphpc) [reply scripts/requirements.txt:2]: https://github.com/unibas-dmi-hpc/SPH-EXA/pull/427/commits/6863a505e0d72c565097b9a9298c99fd5a9e701f

**Comment by sekelle** [comment]
Are there any changes except formatting?

**Comment by jgphpc** [comment]
The only change is in [rollingAverage](
https://github.com/unibas-dmi-hpc/SPH-EXA/blob/3fab1a543f12a829cfb7111585cd614c4240fc2a/scripts/substep_timings.py#L77)

**Comment by jgphpc** [comment]
I will relaunch the CI  jobs after jfrog will be back.

**Comment by sekelle** [comment]
lgtm, please squash the commits

**Pull request merged**